libs/auth/src/env_vars.ts

```diff
@@ -3,8 +3,11 @@ import { assert } from '@votingworks/basics';
 /**
  * Gets an env var and throws an error if it's not set
  */
-export function getRequiredEnvVar(envVarName: string): string {
+export function getRequiredEnvVar<T extends string>(
+  envVarName: T
+): NonNullable<NodeJS.ProcessEnv[T]> {
```

Comment:
This change ensures that the typings in [libs/auth/src/env.d.ts](https://github.com/votingworks/vxsuite/blob/main/libs/auth/src/env.d.ts) still apply when using this helper